modules/gridBidAdapter.js

```diff
@@ -161,66 +165,14 @@ export const spec = {
       userExt = {consent: gdprConsent.consentString};
     }
 
-    if (userId) {
-      if (userId.tdid) {
-        userExt = userExt || {};
-        userExt.eids = userExt.eids || [];
-        userExt.eids.push({
-          source: 'adserver.org', // Unified ID
-          uids: [{
-            id: userId.tdid,
-            ext: {
-              rtiPartner: 'TDID'
-            }
-          }]
-        });
-      }
-      if (userId.id5id && userId.id5id.uid) {
-        userExt = userExt || {};
-        userExt.eids = userExt.eids || [];
-        userExt.eids.push({
-          source: 'id5-sync.com',
-          uids: [{
-            id: userId.id5id.uid
-          }],
-          ext: userId.id5id.ext
-        });
-      }
-      if (userId.lipb && userId.lipb.lipbid) {
-        userExt = userExt || {};
-        userExt.eids = userExt.eids || [];
-        userExt.eids.push({
-          source: 'liveintent.com',
-          uids: [{
-            id: userId.lipb.lipbid
-          }]
-        });
-      }
-      if (userId.idl_env) {
-        userExt = userExt || {};
-        userExt.eids = userExt.eids || [];
-        userExt.eids.push({
-          source: 'identityLink',
-          uids: [{
-            id: userId.idl_env
-          }]
-        });
-      }
-      if (userId.criteoId) {
-        userExt = userExt || {};
-        userExt.eids = userExt.eids || [];
-        userExt.eids.push({
-          source: 'criteo.com',
-          uids: [{
-            id: userId.criteoId
-          }]
-        });
-      }
+    if (userIdAsEids && userIdAsEids.length) {
+      userExt = userExt || {};
+      userExt.eids = [...userIdAsEids];
+    }
 
-      if (userId.digitrustid && userId.digitrustid.data && userId.digitrustid.data.id) {
-        userExt = userExt || {};
-        userExt.digitrust = Object.assign({}, userId.digitrustid.data);
-      }
+    if (userId && userId.digitrustid && userId.digitrustid.data && userId.digitrustid.data.id) {
+      userExt = userExt || {};
+      userExt.digitrust = Object.assign({}, userId.digitrustid.data);
```

Comment:
digitrust has been shut down, I'm curious why you are keeping this in?

Reply:
Fixed